pkg/manager/setconfig.go

```diff
@@ -104,9 +104,31 @@ func (m *Manager) SetNetworkConfig(configName store.ConfigName, updates map[stri
 	deviceConfig.Updated = time.Now()
 	m.ConfigStore.Store[configName] = deviceConfig
 
+	modelName := fmt.Sprintf("%s-%s", deviceConfig.Type, deviceConfig.Version)
+	modelPlugin, ok := m.ModelRegistry[modelName]
```

Comment:
this modelPlugin is actually the device supported models right ? I would go for a more concrete name, otherwise it seems a random plugin.

Reply:
done